src/App.tsx

```diff
@@ -51,6 +52,7 @@ function App() {
                 <ReactCustomScrollbars autoHeight autoHeightMin={'100vh'} autoHide>
                     <Routes>
                         <Route path={'/'} element={<MainPage/>} />
+                        <Route path={'/sponsors'} element={<SponsorsPage/>} />
```

Comment:
You dont need it here, we belove iterating over `menuItems`, so it is enough when it is there